spec/requests/sessions_request_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+describe "Sessions" do
+  describe "GET /new" do
+    it "returns http success" do
+      get "/sessions/new"
+      expect(response).to have_http_status(:success)
+    end
+  end
```

Comment:
missing specs for create and destroy

Reply:
> missing specs for create and destroy

will the other models such as trucks, drivers etc also need specs for create and destroy?

Reply:
yep in another PR though

we want to test all the things as much as is reasonable